src/App.js

```diff
@@ -38,25 +38,24 @@ export default function App() {
 
   return (
     <div>
-      <ToggleBar />
       <Routes>
+        <Route path="*" element={<Navigate to="/futuretrips" replace />} />
```

Comment:
is this route with the * for individual paths for every detailtrip with its own id?

Reply:
not quite.. my route for the DetailTrip-Page is in line 47.
I did this for two reasons:
1. if a user writes an URL by himself which doesn't exist, the user gets back to the "futuretrip" page
2. I don't really have a home-page. This sets the "fututretrip" page as my "home". Otherwise I had problems to route correctly to the DetailTrip-Page.

Hope this is helpful and understandable